db/seeds.rb

```diff
@@ -557,6 +557,28 @@
 
 ##############################################################################
 
+num_of_display_ads = 2 * SEEDS_MULTIPLIER
+
+seeder.create_if_none(DisplayAd) do
+  orgs_in_random_order = Organization.order(Arel.sql("RANDOM()"))
+  orgs = orgs_in_random_order.to_a
+
+  num_of_display_ads.times do
+    DisplayAd.create!(
+      organization: orgs.sample,
+      name: Faker::Lorem.sentence,
+      body_markdown: "This is an external ad",
+      published: true,
+      approved: true,
+      placement_area: "post_comments"
+      # tag_list: Tag.order(Arel.sql("RANDOM()")).first(2).pluck(:name),
+      # type_of: "external", need to random this
```

Comment:
@Ridhwana are these two parts necessary? or should I just leave it out for now?

Reply:
You can leave it out for now: 
- `type_of` it will default to in_house which is good as a first pass.
- without `tag_list` the `display_ad` be targeting all posts which also is fine now. 

We can refine and improve on these seeds as we go along but this already going to be such a big win to have the _some_ billboard seeds.